backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+        revised_at:
+          type: number
+        is_primary_data:
+          $ref: "#/components/schemas/is_primary_data"
+        X_normalization:
+          type: string
+        dataset_assets:
+          type: array
+          items:
+            $ref: "#/components/schemas/dataset_asset"
+        tombstone:
+          type: boolean
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        sex:
+          $ref: "#/components/schemas/ontology_element_array"
+        ethnicity:
+          $ref: "#/components/schemas/ontology_element_array"
+        development_stage:
+          $ref: "#/components/schemas/ontology_element_array"
+        dataset_deployments:
+          type: array
+          items:
+            type: object
+            properties:
+              url:
+                type: string
+        linked_genesets:
+          type: array
+          items:
+            type: string
+        cell_type:
+          $ref: "#/components/schemas/ontology_element_array"
+        cell_count:
+          type: integer
+        X_approximate_distribution:
+          $ref: "#/components/schemas/distribution"
+        schema_version:
+          type: string
+    dataset_preview:
+      type: object
+      required:
+        - id
+        - curator_tag
+        - tissue
+        - assay
+        - disease
+        - organism
+        - tombstone
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tombstone:
+          type: boolean
+    dataset_asset:
+      type: object
+      properties:
+        id:
+          type: string
+        dataset_id:
+          type: string
+        filetype:
+          type: string
+          enum: [H5AD, RDS, CXG]
+        filename:
+          type: string
+        type:
+          type: string
+          enum: [REMIX, ORIGINAL]
+        s3_uri:
+          type: string
+    gene_set:
+      type: object
+      properties:
+        name:
+          type: string
+        id":
+          type: string
+        description:
+          type: string
+        linked_datasets:
+          type: array
+          items:
+            type: string
+    ontology_element:
+      type: object
+      properties:
+        ontology_term_id:
+          type: string
+        label:
+          type: string
+    ontology_element_array:
```

Comment:
To follow the general naming convention, should this be `ontology_elements`?

Reply:
It's not entirely clear to me that there's precedent in our spec for pluralizing the descriptor whenever we append `_array`. This is the only instance where we do that.

Reply:
In that case I second `ontology_elements`

Reply:
Oh I totally misinterpreted the first time around. I was thinking you were both suggesting `ontology_elements_array`. Will change.

Reply:
fixed (`d2586b4d`)